clients/fides-js/src/components/notices/NoticeOverlay.tsx

```diff
@@ -63,13 +59,11 @@ const NoticeOverlay: FunctionComponent<OverlayProps> = ({
   const initialEnabledNoticeKeys = (consent?: NoticeConsent) => {
     if (experience.privacy_notices) {
       // ensure we have most up-to-date cookie vals
-      // TODO (PROD-1792): we should be able to replace parsedCookie with savedConsent
-      const parsedCookie: FidesCookie | undefined = getFidesConsentCookie();
       return experience.privacy_notices.map((notice) => {
         const val = resolveConsentValue(
           notice,
           getConsentContext(),
-          consent || parsedCookie?.consent,
+          consent || savedConsent,
```

Comment:
@gilluminate - flagging this as possibly we want to use your solution instead here

Reply:
I've added back the fallback- the tests were passing before, but the new issue only arose with the added OT migration functionality. So those new tests cover this case where the UI wasn't reflecting those overrides 👍